app/javascript/components/main-menu/helpers.js

```diff
@@ -2,11 +2,13 @@ export const getHrefByType = (type, href, id) => (
   {
     big_iframe: `/dashboard/iframe?id=${id}`,
     // eslint-disable-next-line no-script-url
-    modal: 'javascript:void(0);',
+    modal: '#',
```

Comment:
`javascript:void(0)` is problematic for react,
`#` is problematic for hash router,
*nothing* used to be problematic for IE, but might just work for us, testing :)